addons/mail/static/src/models/messaging_initializer/messaging_initializer.js

```diff
@@ -113,6 +114,8 @@ function factory(dependencies) {
             // failures after channels
             this._initMailFailures(mail_failures);
             discuss.update({ menu_id });
+            // company related data
+            this.messaging.update({ companyName: company_name });
```

Comment:
return `companyName` directly in python, so no rename needed

Reply:
Note you'll need to adapt test_discuss_full when you add it also for the user init messaging